apps/app/src/views/routes/Settings/components/profile/Profile.tsx

```diff
@@ -90,10 +112,29 @@ export const Profile = () => {
     >
       <Container padding={{ base: '0', md: '0' }}>
         <Stack spacing={2}>
-          <Text fontSize="xl" fontWeight="medium">
-            Profile
-          </Text>
+          <Flex minWidth="max-content" alignItems="center" gap="2">
+            <Box>
+              <Text fontSize="xl" fontWeight="medium">
+                Profile
+              </Text>
+            </Box>
+
+            <Spacer />
+
+            <Box>
+              <Button size={'sm'} colorScheme={'brand'} leftIcon={<EditIcon />} onClick={onOpen}>
+                Update Profile
+              </Button>
+            </Box>
+          </Flex>
+          <Modal isOpen={isOpen} onClose={onClose} size="xl">
+            <ModalOverlay />
+            {isOpen && (
```

Comment:
This should only show up if `data` is not null.

Reply:
done